ebook.ddoc

```diff
@@ -75,6 +75,8 @@ _=
 
 _= specification boxes
 BEST_PRACTICE= $(P $(B Best Practices:) $0)
-IMPLEMENTATION_DEFINED=$(P $(B Implementation Defined): $0)
+IMPLEMENTATION_DEFINED=$(P $(B Implementation Defined:) $0)
```

Comment:
The line below this is still `):` though - is that intentional?

Reply:
fixed